CHANGELOG.unreleased.md

```diff
@@ -27,4 +27,6 @@
 
 <!--packages-start-->
 
+- xo-server patch
```

Comment:
This package does not contain any changes, shouldn't we release `@xen-orchestra/backups` and `vhd-lib` instead?